### ISSUE TITLE
Thumbor requests fail

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp.dev#5777]
Haven't had time yet to investigate. The instance id is correct though.

```
{
  "insertId": "r6nq0pork2fknp549",
  "jsonPayload": {
    "cos.googleapis.com/container_id": "3f017a3c997fc40af2086f58bd405ac695ba9df022733af2623eeb16021fbf0c",
    "cos.googleapis.com/stream": "stdout",
    "message": "[9:40:45 PM] [Thumbor] › ✖  error     Failed connecting to thumbor FetchError: request to http://ig-amp-dev-thumbor-n0gc.c.amp-dev-230314.internal:8080/unsafe/https://amp.dev/static/samples/img/image2.jpg%3Foriginal=true failed, reason: getaddrinfo ENOTFOUND ig-amp-dev-thumbor-n0gc.c.amp-dev-230314.internal\n"
  },
  "resource": {
    "type": "gce_instance",
    "labels": {
      "instance_id": "6385818354535104611",
      "zone": "europe-west2-c",
      "project_id": "amp-dev-230314"
    }
  },
  "timestamp": "2021-05-04T21:40:45.692615377Z",
  "labels": {
    "compute.googleapis.com/resource_name": "f237494a92cd"
  },
  "logName": "projects/amp-dev-230314/logs/cos_containers",
  "receiveTimestamp": "2021-05-04T21:40:50.815806749Z"
}
```

**Comment by matthiasrohmer** [comment]
Insert inappropriate language here... I'll have a look tomorrow.

**Comment by sebastianbenz** [comment]
As part of this we should also make `/static/remote/?url=...` fallback gracefully to the actual image.

**Comment by sebastianbenz** [comment]
This problem also affects amppackager. Both, amppackager and thumbor, instances currently cannot be reached even though instanceids are correct ([logs](https://console.cloud.google.com/logs/viewer?project=amp-dev-230314&rif_reserved&minLogLe[%E2%80%A6]%3A23.031073264Z%22%0AinsertId%3D%22pnecxhqub5322thip%22). 

//cc FYI @patrickkettner

**Comment by patrickkettner** [comment]
I shelled into the docker containers for the site to watch for logs, and they were both getting an ENOTFOUND error for the both thumbor and packager. I jumped over to each of the containers, and got their hostname with 

```sh
curl "http://metadata.google.internal/computeMetadata/v1/instance/hostname" \
  -H "Metadata-Flavor: Google"
```

that was slightly different from what we had in the `production.json` file. Specifically, it included the `zone` information (i.e. `us-east1-b`). I [updated the information ](https://github.com/ampproject/amp.dev/commit/1bacd1b6017d4e5b586c17167476b24ecdb0c3a3), and it is no longer failing in production. Will PR the change to get into `future` now
